README.md

```diff
@@ -40,6 +40,10 @@ Check [the official docs](https://docs.datadoghq.com/account_management/api-app-
 Note: Application keys get the same permissions as you. You might want to narrow down the permissions (datadog-service only reads metrics from the API. Check the official docs linked above for more information).
 
 ## If you already have a Keptn cluster running
+Note: Before installing datadog add this repo to your working directory
```

Comment:
This should go under `[1. Install datadog](https://github.com/keptn-sandbox/datadog-service/pull/104/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R47)` because we don't need it for datadog-service (only required for datadog) 

```suggestion
Add datadog helm repo:
```